easybuild/easyconfigs/p/PyTables/PyTables-3.8.0-foss-2022a.eb

```
@@ -0,0 +1,74 @@
+# http://www.pytables.org/usersguide/installation.html
+# updated: Denis Kristak (INUITS)
+
+easyblock = 'PythonBundle'
+
+name = 'PyTables'
+version = '3.8.0'
+
+homepage = 'https://www.pytables.org'
+description = """PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
+ with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
+ NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
+ parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,
+ process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk
```

```suggestion
 processing and searching very large amounts of data. One important feature of PyTables is that it optimizes memory and disk
```

I guess this was wrong upstream a while ago, since fixed, and that hasn't yet been picked up here yet.